frontend/javascripts/oxalis/view/layouting/flex_layout_wrapper.js

```diff
@@ -313,7 +315,6 @@ class FlexLayoutWrapper extends React.PureComponent<Props, State> {
   }
 
   onLayoutChange = () => {
-    sendAnalyticsEvent("change_tracing_layout", { viewMode: this.props.layoutKey });
```

Comment:
I removed the analytics here and moved it into `onAction` as each change to the layout, first triggers an action. This method gets information about what action was done and therefore can better decide whether to send an analytics event.

However, the `onAction` method is not called when the layout is completely rebuild. For that case the event is also send in `rebuildLayout`.